misc/templates/reference/module.cshtml

```diff
@@ -33,11 +33,28 @@
 
 <h1>@Model.Module.Name</h1>
 <p>
+    @if (Model.Module.IsObsolete)
+    {
+        <div class="alert alert-warning">
+        <strong>WARNING: </strong> This API is obsolete
+        </div> 
```

Comment:
Should we include the obsolete message (as you have already implemented that)?

Reply:
We can. The actual message is also listed under the attributes. So if u hover over a method or click into a type/module u can see it there. But i can also include it here.
For Fake i would definitely include it as there are so many obsolete types. But in most APIS there are not that many.